### DISCUSSION TITLE
How to automatic releasing of held messages?

### DISCUSSION THREAD

**Discussion by ffsandi** [postalserver/postal#1990] [Development discussions]
Dear @adamcooke @willpower232 

How to implement this https://github.com/postalserver/postal/issues/144#issuecomment-498635520 on the latest version of postal?

I am facing the same issue. Postal does not automatically resend held messages after 1 hour (hourly limit already 0). But it's held until the next 7 days. Which is not make sense to me. Because postal didn't send any message, and just sitting there doing nothing for next 7 days. 

Thank you for your assistance.

**Comment by willpower232** [comment]
The new installer should still provide `postal console` if you want to run those commands yourself.

Regarding Postals behaviour, see https://github.com/postalserver/postal/issues/1973#issuecomment-1112361245

**Reply by eziobav** [reply]
Can you give me the command to send all held messages? Or where i can found a list of command?
